navlie/utils.py

```diff
@@ -858,6 +858,7 @@ def plot_poses(
     step: int = 5,
     label: str = None,
     linewidth=None,
+    plot_2d=False,
```

Comment:
add to docs below? i.e. what happens when you feed in a 3D pose but set `plot_2d=True`. Birds eye view it seems?